src/rez/tests/test_shells.py

```diff
@@ -145,13 +181,19 @@ def test_rcfile(self):
             self.assertEqual(_stdout(p), "Hello Rez World!")
             os.remove(path)
 
-    @shell_dependent(exclude=["cmd"])
+    @shell_dependent()
     @install_dependent
     def test_rez_env_output(self):
         # here we are making sure that running a command via rez-env prints
         # exactly what we expect.
+        sh = create_shell()
         echo_cmd = which("echo")
-        if not echo_cmd:
+
+        # Certain shells will not find echo.
+        # TODO: If this exception was created for the following shells then
+        #  it is redundant. Can we clarify which platforms this was meant for?
+        has_buildin_echo = sh.name() in ['cmd', 'powershell', 'pwsh']
+        if not echo_cmd and not has_buildin_echo:
```

Comment:
Could we keep this simple and just use whatever the `echo` equivalent is when on Windows? Are there any non-windows shells that wouldn't have `echo`?